src/SALib/sample/morris/__init__.py

```diff
@@ -346,6 +330,59 @@ def _compute_optimised_trajectories(problem, input_sample, N, k_choices,
     return output
 
 
+def define_problem_with_groups(problem: dict) -> dict:
+    """
+    Checks if the user defined the 'groups' key in the problem dictionary.
+    If not, makes the 'groups' key equal to the variables names. In other
+    words, the number of groups will be equal to the number of variables, which
+    is equivalent to don't use groups at all.
+    Parameters
+    ----------
```

Comment:
Change awkward phrasing:

"is equivalent to don't use groups at all."

to

"is equivalent to no groups" (or "groupings"? - I'll leave you to decide)


Additionally, insert a newline between docstring text and parameter list for all functions, e.g:

```
[...] is equivalent to don't use groups at all.

Parameters
----------
problem : dict
```